tests/performance/performancetests.sh

```diff
@@ -273,6 +277,71 @@ runOtherBenchMarkTests()
 	echo >> ${TEST_OUT_DIR}/benchmarks
 }
 
+runMultiplicationKernels()
+{
+	local runner=$1
+	local backend=$2
+	local dataSet=$3
+	local parseMode=$4
+
```

Comment:
add a comment here that reminds the check for the presence of the dataset is assumed to already have happened